### PR TITLE
Implemented few more instructions for Apt

### PR DESCRIPTION
**Pull request by BSG-75**
Which are `EA_PushRegister`, `EA_PushConstantWord`, `EA_CallFuncPop`, `StrictEqual`.

### CODE REVIEW

File: src/OpenSage.Game/Gui/Apt/ActionScript/InstructionCollection.cs [hunk 1]
```diff
@@ -422,6 +422,20 @@ public void Parse()
                     case InstructionType.Var:
                         instruction = new Var();
 
+                        break;
```
Comment (feliwir) [inline src/OpenSage.Game/Gui/Apt/ActionScript/InstructionCollection.cs]: Remove the empty line above the break please

File: src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Logic.cs [hunk 2]
```diff
@@ -29,6 +59,7 @@ public override void Execute(ActionContext context)
     }
 
     //Pop two values A,B from stack and check if B is greater A (reverse stack order)
+    // ^ Did us forget to edit comment after having copied it?
```
Comment (feliwir) [inline src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Logic.cs]: Yes, please change it :+1:

File: src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Function.cs [hunk 3]
```diff
@@ -292,4 +292,27 @@ public override void Execute(ActionContext context)
             FunctionCommon.ExecuteFunction(funcName, args, obj, context);
         }
     }
+
+    /// <summary>
+    /// (Just guessing) Call an function which its name is on the stack. Function arguments are also popped from the stack
+    /// </summary>
+    public sealed class CallFunctionPop : InstructionBase
+    {
+        public override InstructionType Type => InstructionType.EA_CallMethodPop;
```
Comment (feliwir) [inline src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Function.cs]: Wrong Type here

File: src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Function.cs [hunk 4]
```diff
@@ -292,4 +292,27 @@ public override void Execute(ActionContext context)
             FunctionCommon.ExecuteFunction(funcName, args, obj, context);
         }
     }
+
+    /// <summary>
+    /// (Just guessing) Call an function which its name is on the stack. Function arguments are also popped from the stack
```
Comment (feliwir) [inline src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Function.cs]: Should be correct. Remove the guessing note please :)

**Comment by feliwir** [comment]
@BSG-75 do you want to update this otherwise i'll do it this evening.

**Comment by BSG-75** [comment]
@feliwir sorry, was a bit busy lately, I've fixed them now.

**Pull request merged**